datalad/cmdline/helpers.py

```diff
@@ -31,11 +32,25 @@ def __call__(self, parser, namespace, values, option_string=None):
             # lets use the manpage on mature systems ...
             try:
                 import subprocess
+                # extract version field from datalad manpage if present, swallow_error_logs
+                man_th = subprocess.check_output(
+                    ['zgrep', '\\.TH', '/usr/share/man/man1/datalad.1.gz'],
+                    stderr=subprocess.STDOUT)
+                man_version = man_th.split(' ')[5].strip(" '\"\t\n")
+
+                # extract current datalad version
+                datalad_version = '.'.join(
+                    __version__.split('.')[:3]
+                ).strip(" '\"\t\n")  # e.g 2.0.3.dev84 ~> 2.0.3
```

Comment:
and don't strip any 'dev84' off -- that is the point, that if we are using development version (from git) we shouldn't use system wide manpages.